tony-core/src/main/java/com/linkedin/tony/Framework.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.linkedin.tony;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+
+import com.linkedin.tony.tensorflow.TonySession;
+import com.linkedin.tony.util.Utils;
+
+public class Framework {
```

Comment:
add documentation for this class. `Framework` is a generic name, I don't know a better name either, so better add more documentation explaining

Reply:
Got it.